src/Apps/Feature/Components/FeatureSet/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./FeatureSet"
```

Comment:
I think we try to avoid these extra `index` files, but @damassi will know more about convention.

Reply:
I personally like it so that it's simple to sprawl out into sub-components when needed and expose what you in a more explicit manner, but yeah I can see the other side of this coin. Lemme know @damassi

Reply:
`index` files make a lot of sense as a funnel in lib code, but unless the folder (in this case `FeatureSet`) will contain numerous files its best to keep things as flat as possible. (It's also nice to (not) have five tabs open in VSCode that are all named `index` 😄)

There are some exceptions to this tho:

Say you have this kind of folder structure:

```
Apps
  AppName 
     Routes
        RouteName
           Components
```
It can make sense to at the `AppName` level to have an `index` file that then coordinates various exports for the sub-app namespace. 

For individual components tho the rule is: 
- One file, flat in Components folder
- Two plus files, feel free to create folder to contain related items and then an index file is ok, but not until there's a logical grouping of items necessary.